tests/rbd/test_concurrent_writes_rbd_persistent_writeback_cache.py

```diff
@@ -0,0 +1,232 @@
+    try:
+        config_level, entity = get_entity_level(cfg)
+        pool, image = cfg["image_spec"].split("/")
+
+        cache.rbd.toggle_image_feature(
+            pool, image, feature_name="object-map,exclusive-lock", action="enable"
+        )
+        cache.configure_pwl_cache(
+            cfg["rbd_persistent_cache_mode"],
+            config_level,
+            entity,
+            cfg["cache_file_size"],
+        )
+
+        mount_path_1 = f"/tmp/mnt_{cache.rbd.random_string(len=5)}/file_1"
+        io_config = {
+            "rbd_obj": cache.rbd,
+            "client": client,
+            "size": "10G",
+            "do_not_create_image": True,
+            "config": {
+                "file_size": "1G",
+                "file_path": [mount_path_1],
+                "image_spec": [cfg["image_spec"]],
+                "operations": {
+                    "fs": "ext4",
+                    "io": False,
+                    "mount": True,
+                    "nounmap": True,
+                    "device_map": True,
+                },
+                "skip_mkfs": False,
+            },
+        }
+        krbd_io_handler(**io_config)
+
+        mount_path_2 = f"/tmp/mnt_{cache.rbd.random_string(len=5)}/file_2"
+        io_config["config"]["operations"]["device_map"] = False
+        io_config["config"]["file_path"] = [mount_path_2]
+        io_config["config"]["operations"]["nounmap"] = True
+        io_config["config"]["skip_mkfs"] = True
+
+        krbd_io_handler(**io_config)
+
+        with parallel() as p:
+            p.spawn(
+                run_fio,
+                client_node=cache.rbd.ceph_client,
+                filename=mount_path_1,
+                size="1G",
+            )
+            p.spawn(
+                run_fio,
+                client_node=cache.rbd.ceph_client,
+                filename=mount_path_2,
+                size="1G",
+            )
+            try:
+                cache.check_cache_file_exists(
+                    cfg["image_spec"],
+                    cfg["fio"].get("runtime", 120),
+                    **cfg,
+                )
+                log.info(
+                    "PWL Cache file created with exclusive lock for concurrent writes..."
+                )
+            except PWLException as error:
+                log.error(f"{error} in entire FIO execution...")
+                raise Exception(error)
+
+        # Verification step, check if both file_1 and file_2 are present in both the
+        # mount points
+
+        out, err = cache.rbd.exec_cmd(
+            cmd=f"ls -l {mount_path_1.rsplit('/', 1)[0]}", all=True
+        )
+        out_2, err_2 = cache.rbd.exec_cmd(
+            cmd=f"ls -l {mount_path_2.rsplit('/', 1)[0]}", all=True
+        )
+        if not (err or err_2):
+            log.info(f"Files in first mount point: {out}")
+            log.info(f"Files in second mount point: {out_2}")
+            if out == out_2 and "file_1" in out and "file_2" in out:
```

Comment:
`file_2` should in `out_2` Is it?

Reply:
No, both mount points should contain both the files, that's the verification to say that data written in both mount points are in sync with each other and also that they have been stored in the corresponding rbd image.
This is what we discussed right?